backend/tracim_backend/lib/utils/dict_parsing.py

```diff
@@ -0,0 +1,120 @@
+import re
+import typing
+
+from marshmallow import ValidationError
+from marshmallow.validate import Regexp
+
+
+def validate_simple_dict(dict_: typing.Dict) -> None:
```

Comment:
Is this method used somewhere? Shouldn't we create a more precise type annotation for the schema dicts instead?

Reply:
This is used to validate user_config. user_config just not include complexe structure. I just moved it here because there is too much utils in utils.